### PR TITLE
Added nuget package information to csproj file.

### PR DESCRIPTION
**Pull request by AraHaan**
Also added some more frameworks as well.

### CODE REVIEW

**Comment by AraHaan** [comment]
@JustNrik I just realized that I forgot to document the WaitForProcessExit property that was merged in my first pull request.

File: NetUtilities/NetUtilities.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netcoreapp2.0;netcoreapp2.1;netcoreapp2.2;net45;net46;net47</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.0;netcoreapp2.1;netcoreapp2.2;net47</TargetFrameworks>
```
Comment (JustNrik) [inline NetUtilities/NetUtilities.csproj]: C:\Program Files\dotnet\sdk\2.2.100-preview2-009404\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(199,5): error NU5026: The file 'C:\projects\netutilities\NetUtilities\bin\Any CPU\Release\net47\NetUtilities.dll' to be packed was not found on disk. [C:\projects\netutilities\NetUtilities\NetUtilities.csproj]

File: NetUtilities/NetUtilities.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netcoreapp2.0;netcoreapp2.1;netcoreapp2.2</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.0;netcoreapp2.1;netcoreapp2.2:net47</TargetFrameworks>
```
Comment (JustNrik) [inline NetUtilities/NetUtilities.csproj]: nuget : Invalid restore input. Invalid target framework 'unsupported'. Input files: C:\projects\netutilities\NetUtilities\NetUtilities.csproj.
At line:1 char:1
+ nuget restore
+ ~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (Invalid restore...ilities.csproj.:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
Command executed with exception: Invalid restore input. Invalid target framework 'unsupported'. Input files: C:\projects\netutilities\NetUtilities\NetUtilities.csproj.

**Comment by JustNrik** [comment]
I will not target any other platform yet. .NET Standard 2.0 class libraries are perfectly supported by .NET Core 2.2 and .NET Framework 4.7.2

Feel free to pr again if you find out how to make it pass appveyor checks

**Pull request merged**